docs/price_oracle_intro.md

```diff
@@ -1,322 +1,222 @@
-    from a consensus engine module, e.g. the Tendermint node;
-- Skill `valory/abstract_abci:0.1.0`: it provides a
-    scaffold handler for ABCI requests. It is an abstract skill.
-- Skill `valory/abstract_round_abci:0.1.0`: it
-    implements an ABCI handler and provides
-    useful code abstractions for creating round-based
-    replicated state machines, based on the ABCI protocol
-    (e.g. `Period`, `AbstractRound`). It is an abstract skill.
-
-Moreover, it has the following specific components to implement the particular case of the price oracle agent service:
-
-- `valory/price_estimation_abci`: it implements the round-based
-    ABCI application for price estimation of a cryptocurrency,
-    with a finalization step over an Ethereum chain.
-
-
-## Price Estimation as a Proof of Concept
-
-The ABCI-based replicated FSM (`AbciApp`) used for price estimation consists
-of several smaller modules, each of which is an `AbciApp` of its own. Each of
-them is a `Skill`, which implies that they operate independently of each other,
-however they can be combined to create a larger `AbciApp`, provided that the
-developer specifies the required transition mapping to connect these FSMs.
-This modularity allows a developer to use a subset of these skills in different
-contexts, potentially in combination with skills they themselves define, to
-create another composite `AbciApp` that performs according to their particular
-needs. Specifically, the `PriceEstimationAbciApp` is created by combining the
-following parts:
-
-- `AgentRegistrationAbciApp`
-- `SafeDeploymentAbciApp`
-- `OracleDeploymentAbciApp`
-- `PriceAggregationAbciApp`
-- `TransactionSubmissionAbciApp`
-- `ResetPauseABCIApp`
-
-### The `AgentRegistrationAbciApp`
-
-This `AbciApp` implements the registration of agents to partake in the behaviour
-scheduled in subsequent rounds.
-
-0. `RegistrationStartupRound` <br/>
-
-1. `RegistrationRound` <br/>
-   In this round registrations from AEAs to join the period are accepted, up to
-   a configured maximum number of participants (in the demo, this limit is 4);
-   once this threshold is hit ("registration threshold"), the round is finished.
-
-2. `FinishedRegistrationRound` <br/>
-   A round that signals agent registration was successful, but service contracts has not been already deployed.
-
-3. `FinishedRegistrationFFWRound` <br/>
-   A round that signals agent registration was successful, and the service contracts are already deployed.
-
-
-### The `SafeDeploymentAbciApp`
-
-This `AbciApp` implements the deployments of a Gnosis safe contract, which is
-a multisig smart contract wallet that requires a minimum number of people to
-approve a transaction before it can occur. This assures that no single agent
-can compromise the funds contained in it.
-
-0. `RandomnessSafeRound` <br/>
-   Some randomness is retrieved to be used in a keeper agent selection. In
-   particular, agents individually request the latest random number from
```

Comment:
Has this been completely removed or just moved?

Reply:
This has been moved to the technical details section (in another file). The overall description of demos should follow:
Intro - Architecture - Running - Interacting - Technical Details - (FSMs)
In this case, Technical details and FSMs are in separate files.